test/integration/targets/ios_logging/tests/cli/basic.yaml

```diff
@@ -127,3 +127,20 @@
       - 'result.changed == true'
       - '"no logging console" in result.commands'
       - '"no logging buffered" in result.commands'
+
+- name: Configure logging trap
+  ios_logging:
+    dest: host
+    name: 172.16.0.1
+    facility: local6
+    level: warnings
+    trap: warnings
+    state: present
+  register: result
+
+- assert:
+    that:
+      - 'result.changed == true'
+      - '"logging host 172.16.0.1" in result.commands'
+      - '"logging facility local6" in result.commands'
+      - '"logging trap warnings" in result.commands'
```

Comment:
Please add extra tests that do:

1) Run the same task again and assert that it's idempotent (`.changed == false`, `.commands` is empty)

2) Change the logging level and ensure that the module returns changed.

Reply:
Hi @gundalow 
Sorry, could you give me an advice? I can't understand what should I add.
My current state on screenshot.
![screenshot](https://user-images.githubusercontent.com/13959303/35059494-3897921e-fbb3-11e7-9072-022d9608b90c.png)

Reply:
Yup, that looks right

Reply:
Ignore the comment about changing logging level